### ISSUE TITLE
Rocminfo Fails

### ISSUE THREAD

**Issue by BemusedCat** [ROCm/rocminfo#38]
I tried everything to run rocm-tensorflow but unable to do so .
Tried everything but nothing works
My rocminfo
`ROCk module is loaded`
`Unable to open /dev/kfd read-write: Bad address`
`abhigyan is member of render group`
`hsa api call failure at: /src/rocminfo/rocminfo.cc:1142`
`Call returned HSA_STATUS_ERROR_OUT_OF_RESOURCES: The runtime failed to allocate the necessary resources. This error may also occur when the core runtime library needs to spawn threads or create internal OS-specific events.`
`

**Comment by fxkamd** [comment]
Sounds like an installation or configuration problem. Can you post dmesg output and "ls -l /dev/kfd" for a start?

**Comment by skeelyamd** [comment]
The problem is permission to access the device driver interface file as indicated here "Unable to open /dev/kfd".  The permissions /group membership needed depends on your specific environment.  Running 'ls -l /dev/kfd' will show the owning group and it's permissions.  Ensure that you are a member of that group.

**Comment by BemusedCat** [comment]
This is the output of 'ls -l /dev/kfd'.
`crw-rw-rw- 1 root render 237, 0 Aug 25 00:11 /dev/kfd`


When I run tensorflow I get this error.
`2020-08-25 02:39:00.789159: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library` `'libhip_hcc.so'; dlerror: libhip_hcc.so: cannot open shared object file: No such file or directory`
`2020-08-25 02:39:00.789209: F tensorflow/stream_executor/lib/statusor.cc:34] Attempting to fetch value instead of handling error Failed precondition: Could not load dynamic library 'libhip_hcc.so'; dlerror: libhip_hcc.so: cannot open shared object file: No such file or directory`
`Aborted (core dumped)`

**Comment by skeelyamd** [comment]
That looks like HIP is not installed.  Did you install the complete set of rocm packages?  What is in /opt/rocm/lib and /opt/rocm/bin?

**Comment by BemusedCat** [comment]
Yes I did installed.
I even fresh installed ubuntu and again did everything.

**Comment by zyzzyxdonta** [comment]
Hello,
are there any news on this issue? 

I ran into the same problem using the rocm/dev-ubuntu-18.04 docker image inside a CI machine. It took me quite a bit of digging to find this because `rocminfo` is called by `rocm_agent_enumerator` which, instead of reporting this error when `rocminfo` fails, just gets stuck.

I noticed another thing, though I'm not sure where to report this: `rocm_agent_enumerator` uses `lspci` which is not installed inside the docker image.
